scripts/build-container-image.sh

```diff
@@ -47,22 +65,17 @@ git_get_current_commit_short_hash() {
   git rev-parse --short HEAD
 }
 
-git_get_current_branch() {
-  git rev-parse --abbrev-ref HEAD
-}
-
 get_date() {
   # Should this be current date, or latest commit date?
   date +%Y%m%d
 }
 
-get_image_tag() {
-  echo "$(git_get_current_branch)_$(get_date)_$(git_get_current_commit_short_hash)"
-}
-
 cleanup_old_tags() {
```

Comment:
```suggestion
cleanup_old_tags() {
  declare -a branch_tags=()
  declare -a all_tags=()
  declare -a sorted_existing_tags=()
```
Start by declaring your arrays.